src/components/BusinessHoursDay.vue

```diff
@@ -195,6 +197,14 @@ export default {
       el.style.visibility = binding.value ? 'visible' : 'hidden';
     }
   },
+  mounted: function() {
```

Comment:
```suggestion
  mounted: function () {
```

Reply:
@jefferson-aloware  thanks for the review, changes are applied 👍